CGCharter-1727386911.html

```diff
@@ -255,36 +260,177 @@ <h2 id="decision">
-      The individual who receives the most votes, no two from the same
-      organisation, is elected chair. In case of a tie, RFC2777 is used to
-      break the tie. An elected Chair may appoint co-Chairs.
-      </li>
-    </ol>
     <p>
-      Participants dissatisfied with the outcome of an election may ask the
-      Community Development Lead to intervene. The Community Development Lead,
-      after evaluating the election, may take any action including no action.
+      The Community Group participants elect (and/or other chairs appoint) no
+      less than One (1) and no more than Three (3) Chairs for the group at any
+      given time. The W3C Team Contact of the group SHOULD NOT be one of the
+      chairs at any point. Participation as Chair is afforded to the specific
+      individuals elected to those positions, and a participant&quot;s seat MUST
+      NOT be delegated to any other person.
+    </p>
+    <p>      
+      For most Chair nominees, the primary affiliation is their employer and
+      will match their affiliation in the W3C database. For contractors and
+      independents, this will normally be their contracting company or their
+      independent status; in some cases (e.g. where a consultant is consulting
+      for only one organization) this may be the organization for whom the
+      nominee is consulting.
+      <strong>Chair nominees, elected chairs, and appointed chairs MUST have
+      unique affiliations publicly stated at time of nomination, election (if it
+      has changed since last disclosure), or appointment.</strong>
+      Publicly stating additional or secondary affiliations is not required but
+      recommended. An organization with which one or more participants are
+      affiliated MAY submit one ballot that ranks candidates in their preferred
+      order as an organization as an informational exhibit; individual
+      participants should still vote explicitly, whether concurring or
+      dissenting with such an exhibit.
     </p>
+
+    <h2 id="choosing-chairs">
+      Chair Selection
+    </h2>
+    <ul>
+      <li>
+        At any given time there may be up to three co-chairs, each holding one
+        seat. Each seat defines a 2 year cycle of service.
+      </li>
+      <li>
+        In the first election after ratification of this charter, all seats will
+        be up for election. Thereafter, in each year, a single election will be
+        held to fill any vacant seats.
+      </li>
+      <li>
+        In the case of interim vacancy, the remaining chairs may appoint a
+        co-chair for each open seat, hold an election for the same, or wait
+        until the next election, at their discretion. If the chairs do not take
+        any action, the seat will automatically be up for election in the next
+        cycle. Any such interim appointments or elections shall hold the seat
+        until the end of its natural cycle.
```

Comment:
the Solid CG charter that I modeled this on had _appointments_ for the rest of the existing term and _elections_ only at regular times, I believe-- maybe I should make that more explicit here.

Reply:
I had to read the paragraph a few times but did come to @bumblefudge's conclusion: an interim person in a position (whether appointed or elected) lasts until the end of the natural cycle of the seat the person was appointed into. So in @ThisIsMissEm's case, yes, the person being appointed or elected would hold the seat for 18 months instead of 24. Not sure it needs rewording as-is.